docker-hub/accounts.md

```diff
@@ -1,41 +1,35 @@
 ---
 description: Your Docker ID
 keywords:
-- Docker, docker, trusted, sign-up, registry, accounts, plans, Dockerfile, Docker
-  Hub, docs,  documentation
-menu:
-  main:
-    parent: mn_pubhub
-    weight: -90
+- Docker, docker, trusted, sign-up, registry, accounts, plans, Dockerfile, Docker Hub, docs,  documentation
 title: Your Docker ID
```

Comment:
"Docker ID and Docker Hub account"
(Same comment as in several others - I know the original title isn't great, so let's improve it.)